burotel/MANIFEST.in

```diff
@@ -0,0 +1,3 @@
+graft indico_burotel/template_overrides
```

Comment:
I think there's more missing in the manifest than just this one. `indico_burotel/static` for example.

Reply:
right, I wrongly assumed I don't have to add anything else there since the manifest didn't exist before